package.json

```diff
@@ -28,7 +29,7 @@
     "start-stepper": "cd packages/stepper && yarn start",
     "start-forms": "cd packages/forms && yarn start",
     "start-theme": "cd packages/theme && yarn start",
-    "publish": "lerna publish --force-publish=*",
+    "publish": "lerna publish from-package --yes --no-verify-access",
```

Comment:
*Details*
`from-package`: doesn't bump version, but published the packages that are not already published
`--yes`: no prompt
`--no-verify-access`: this is buggy, it ask npm if user has the push rights on the package to publish, but npm response can be wrong